packages/typespec-vscode/src/vscode-cmd/create-tsp-project.ts

```diff
@@ -425,24 +383,25 @@ async function validateTemplate(
     }
   }
 
-  const validateResult = await client.validateInitProjectTemplate(info.template);
-  if (!validateResult) {
-    logger.warning("Template validation failed. Please check the previous log for details.", [], {
-      showOutput: true,
-      showPopup: true,
-    });
-    const cont = await vscode.window.showQuickPick(["Yes", "No"], {
-      canPickMany: false,
-      placeHolder:
-        "Template validation failed. Do you want to continue? Detail log can be found in the Output window.",
-      ignoreFocusOut: true,
-      title: TITLE,
-    });
-    if (cont !== "Yes") {
-      logger.info("Creating TypeSpec Project cancelled due to template validation failure.");
-      return false;
-    }
-  }
+  // TODO: Why is this here again?
```

Comment:
call validate in compiler to do further validation (check json schema). you can check validateTemplate in init.ts.